src/main/java/org/broadinstitute/consent/http/service/MetricsService.java

```diff
@@ -217,6 +217,10 @@ public String getHeaderRow(Type type) {
       .stream()
       .filter(e -> Objects.nonNull(e.getFinalVote()))
       .findFirst();
+
+    if(election.isEmpty()) {
+      election = electionList.stream().findFirst();
+    }
```

Comment:
This took me a minute to sort through in my head. I think that what we're looking for in `electionList` is either the first one with a final vote or any other election at all, regardless of final vote status. I think we can clean up the lambda to do both things in one pass. If we first sort on final vote (treating nulls as lower in sort order) instead of filtering, then `findFirst` works in both of our cases. I _think_ this might work (have not tested, and there are further optimizations to be done here):

```
    Optional<Election> election = electionList
      .stream()
      .sorted(Comparator.nullsLast(Comparator.comparing(Election::getFinalVote)))
      .findFirst();
```

Reply:
I changed the `sorted` call a bit since NullPointerExceptions can still be thrown within `Comparator.comparing`

```
Optional<Election> election = electionList
      .stream()
      .sorted(Comparator.comparing(Election::getFinalVote, Comparator.nullsLast(Comparator.naturalOrder())))
      .findFirst();
```

I removed the outer `nullsLast` call since the initial DAO call and the subsequent filters ensure that the electionList parameter will not have null elements or be null itself (it'll just be an empty list)